docs/src/charm/howto/openstack.md

```diff
@@ -0,0 +1,104 @@
+# Integrating with OpenStack
+
+This guide explains how to integrate {{product}} with the OpenStack cloud
+platform.
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- An [OpenStack][openstack] cloud environment.
+- Juju set-up as the deployment tool for Openstack. Please refer to
+  [Juju and Openstack][juju-openstack] documentation for more information.
+- An OpenStack [project][project] with the necessary permissions to deploy
```

Comment:
nit: We already mentioned the we need an Openstack environment in the first point. Maybe we can remove one of these two.